meson.build

```diff
@@ -0,0 +1,364 @@
+project('abseil-cpp', 'cpp',
+  version: '20200225.2',
+  license: 'Apache-2.0')
+
+cc = meson.get_compiler('c')
+
+c_flags = [
```

Comment:
Optional: `cc_flags` or `cpp_flags` might be more appropriate.

Reply:
Done (cpp_flags).